src/styles/respec.css.js

```diff
@@ -218,34 +218,46 @@ aside.example .marker > a.self-link {
   color: inherit;
 }
 
+.header-wrapper {
+  display: flex;
+  align-items: center;
```

Comment:
Breaks when heading wraps across multiple lines, but maybe ok.

Reply:
Can live with that.